### DISCUSSION TITLE
Using renovate to rebaseLabel to rebase non-renovate PRs

### DISCUSSION THREAD

**Discussion by roded** [renovatebot/renovate#21527] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

github.com

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

Hello,
Is it possible to use renovate's rebaseLabel to auto-rebase PRs which were not created by renovate?
Thank you.

### What you tried so far.

N/A

**Comment by rarkins** [comment, marked as answer]
No, not possible
